content/en/docs/tasks/administer-cluster/memory-manager.md

```diff
@@ -123,6 +130,24 @@ In the case of the `BestEffort` or `Burstable` pod, the `Static` Memory Manager
 the default topology hint as there is no request for the guaranteed memory,
 and does not reserve the memory in the internal [NodeMap][2] object.
 
+This policy is only Supported on Linux.
+
+#### BestEffort policy {#policy-best-effort}
+
+{{< feature-state for_k8s_version="v1.32" state="alpha" feature_gate_name="WindowsCPUAndMemoryAffinity" >}}
+
+This policy is only Support on Windows.
+
+On Windows, NUMA node assignment works differently than Linux.  There is no mechanism
+to ensure that Memory access only comes from a specific NUMA node, instead the Windows
+internals will select the most optimal NUMA node based on the CPU assignment and it is
+possible that Windows might use other nodes if required or deemed optimal.  For this
+reason this policy is best effort.  
+
+The policy does track the amount of memory available and requested through the internal
+[NodeMap][2] to make a best effort at ensuring that memory enough memory is available on
+a NUMA node before making the hint to assign it to the NODE.  This means that in most cases, this should function as expected.
```

Comment:
Please consider revise the looooooooong sentence into shorter ones, in simple English.
The text is for user to understand anyway.

Reply:
I've attempted to simplify with shorter sentences.  PTAL